src/client/src/routes/NotificationRoute.tsx

```diff
@@ -14,28 +13,48 @@ interface State {
   message: Message | null
 }
 
+const debugTrade: Trade = {
```

Comment:
Wouldn't it be better and easier to use elsewhere if one extracted the strings defined in this variable to an enum instead?